flyway/postgre/migrations/sql/V1_3__Create_lesson2_tables.sql

```diff
@@ -0,0 +1,87 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+DROP TABLE IF EXISTS sso_user_roles;
+DROP TABLE IF EXISTS sso_organization_user;
+DROP TABLE IF EXISTS sso_user_profile;
+DROP TABLE IF EXISTS sso_user;
+DROP TABLE IF EXISTS sso_roles;
+DROP TABLE IF EXISTS sso_organization;
```

Comment:
Không nên xóa bảng vì có thể gây mất dữ liệu. Chỉ cần câu lệnh CREATE TABLE IF NOT EXISTS thì sẽ không bị lỗi khi bảng đã tồn tại.
```suggestion
```